ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-summary/index.scss

```diff
@@ -23,12 +23,36 @@
   &__action {
     @include H7;
 
-    text-transform: uppercase;
     color: var(--oslo-gray);
     padding: 3px 8px;
     border: 1px solid var(--oslo-gray);
     border-radius: 4px;
-    display: inline-block;
+    display: inline-flex;
+    align-items: center;
+
+    &__name {
+      text-transform: uppercase;
+    }
+
+    .info-tooltip {
+      margin-left: 4px;
+
+      &__tooltip-container {
+        margin-bottom: -3px;
+      }
+    }
+
+    &__contract-address {
+      margin-right: 4px;
+    }
+
+    &__contract-address-btn {
+      background: none;
+      border: none;
+      padding: 0;
+      margin-right: 4px;
```

Comment:
We probably want `margin-inline-end` here.

Reply:
fixed 👍